pvlib/test/test_atmosphere.py

```diff
@@ -128,6 +128,21 @@ def test_first_solar_spectral_correction_ambiguous():
         atmosphere.first_solar_spectral_correction(1, 1)
 
 
+def test_first_solar_spectral_correction_range():
+    with pytest.warns(UserWarning):
```

Comment:
We've started testing the content of the warning messages. Example [here](https://github.com/pvlib/pvlib-python/blob/a349bea38d928696671026dc1e126d44d8d2e529/pvlib/test/test_modelchain.py#L600). OK if you don't want to add, but I thought you'd like to see how it's being done.